File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.java
class Solution {
    public int[] solution(String[] wallpaper) {
        int[] answer = {51, 51, -1, -1};
        for(int i=0; i<wallpaper.length; i++){
            String[] data = wallpaper[i].split("");
            for(int j=0; j<data.length; j++){
                if("#".equals(data[j])){
                    answer[0] = answer[0] < i ? answer[0] : i;
                    answer[1] = answer[1] < j ? answer[1] : j;
                    answer[2] = answer[2] > i ? answer[2] : i+1;
                    answer[3] = answer[3] > j ? answer[3] : j+1;
                }
            }
        }
        return answer;
    }
}

File: 프로그래머스/1/161990. 바탕화면 정리/바탕화면 정리.py
def solution(wallpaper):
    answer = [51, 51, -1, -1]
    for i in range(len(wallpaper)):
        data = wallpaper[i]
        for j in range(len(data)):
            if data[j] == "#":
                answer[0] = answer[0] if answer[0] < i else i
                answer[1] = answer[1] if answer[1] < j else j
                answer[2] = answer[2] if answer[2] > i else i+1
                answer[3] = answer[3] if answer[3] > j else j+1
    return answer